docs/reference/technology/psa/psa.md

```diff
@@ -32,11 +32,14 @@ The Mbed implementation of PSA supports the following platform types:
 
 - Asymmetric Multiprocessing (AMP) systems: Multicore ARMv7-M targets (for example, PSoC6 featuring CM4 and CM0+ cores). On these targets, one of the cores is dedicated to PSA use only and implements SPE. The Mbed implementation of PSA provides PSA API proxy implementation on a nonsecure core, which redirects execution to the SPE.
 
-- ARMv8-M: Generation of ARM processors featuring TrustZone-M architecture. PSA support for this platforms is in final stages of development and will be added to the list of platforms supported by the Mbed implementation of PSA shortly.
+- ARMv8-M: Generation of ARM processors featuring TrustZone-M architecture. PSA support for these platforms is based on a *specialized* [TrustedFirmware-M](https://www.trustedfirmware.org) implementation.
 
 ### RoT services
 
 The Mbed implementation of PSA provides the following services:
 
 - PSA [RoT](../introduction/glossary.html) internal storage.
```

Comment:
@alzix, do you want to leave these links in or did you want to put a link to the API > Security page?
The RoT link points to the glossary. Where should it be pointing to?